js/webidl2.js

```diff
@@ -582,7 +582,7 @@
                 return ret;
             }
             else if (consume(ID, "stringifier")) {
-                ret.stringifier = true;
+                ret.stringifier = true;-
```

Comment:
this is a copy of webidl2.js from the its repo; I suggest NOT to review changes in this file

(ideally, we would not copy but import webidl2.js; but that's another story)